plugin/src/test/java/org/jenkinsci/plugins/workflow/cps/CpsScriptTest.java

```diff
@@ -92,4 +92,124 @@ public void evaluateShallSandbox() throws Exception {
         r.assertLogContains("Scripts not permitted to use method groovy.lang.Script run java.io.File java.lang.String[]", b);
     }
 
+    @Test public void methodTooLargeExceptionFabricated() throws Exception {
+        // Fabricate a MethodTooLargeException which "normally" happens when evaluated
```

Comment:
Is there a reason to keep this test case, given the “realistic” test below?

Reply:
I think yes, this fabricated test is about coded reaction to the particular exception, which may be or not be the one thrown in realistic test below (if CPS VM does at all break such way there and with that amount of threads).